bundles/org.palladiosimulator.dataflow.confidentiality.analysis/src/org/palladiosimulator/dataflow/confidentiality/analysis/sequence/pcm/PCMNodeCharacteristicsCalculator.java

```diff
@@ -25,8 +25,8 @@
 import org.palladiosimulator.pcm.usagemodel.UsageScenario;
 
 public class PCMNodeCharacteristicsCalculator {
-	private final Logger logger = Logger.getLogger(PCMNodeCharacteristicsCalculator.class);
-    private final Entity node;
+	private Logger logger = Logger.getRootLogger();
+    private EObject node;
```

Comment:
Is this a merge error?

Reply:
Yeah, it seems i screwed up the rebase, but it should be fixed now